configs/samples/pjsip.conf.sample

```diff
@@ -1509,9 +1509,9 @@
 
 
 ; MODULE PROVIDING BELOW SECTION(S): res_pjsip_pubsub
-;=============================RESOURCE-LIST===================================
 ; See https://wiki.asterisk.org/wiki/pages/viewpage.action?pageId=30278158
```

Comment:
This is https://docs.asterisk.org/Configuration/Channel-Drivers/SIP/Configuring-res_pjsip/Resource-List-Subscriptions-RLS/